logger/event_logger.go

```diff
@@ -27,7 +27,7 @@ func (l *eventLogger) Close() error {
 	return l.eventlog.Close()
 }
 
-func (l *eventLogger) Print(level telegraf.LogLevel, _ time.Time, prefix string, args ...interface{}) {
+func (l *eventLogger) Print(level telegraf.LogLevel, _ time.Time, prefix string, attr map[string]interface{}, args ...interface{}) {
```

Comment:
Won't there be some linter warning about unused var? Maybe better do this for the time being:
```suggestion
func (l *eventLogger) Print(level telegraf.LogLevel, _ time.Time, prefix string, _ map[string]interface{}, args ...interface{}) {
```